### ISSUE TITLE
Ender Dragon and Fountain not spawning

### ISSUE THREAD

**Issue by Soulbrainer** [The-Nexus-Project/Limitless-3#48]
Updated to 1.8.0 on server and made sure to follow instructions as well as delete DIM1 file to reset End. Still no Ender Dragon or Fountain Spawn

- Pack Version: 1.8.0
- On a server?: Yes

**Comment by Gbergz** [comment]
Yeah no matter how hard I try, I can't reproduce this issue
![image](https://user-images.githubusercontent.com/26634554/100546071-98192a00-325f-11eb-8d6b-e4ef99b6725d.png)

**Comment by Soulbrainer** [comment]
Maybe its because we didnt have one before we updated?

**Comment by grim3212** [comment]
Adding what worked for me. I was running into the same issue even after updating to 1.8.0 where even deleting the DIM1 folder the fountain and dragon weren't generating. After a bunch of trial and error I finally was able to get the dragon to spawn with the fountain.

Here's what worked for me:
- Stop server
- Backup everything just in case
- Delete the DIM1 folder
- Open the `level.dat` with an NBT editor. I was using NBTExplorer
- Make the Data/WorldGenSettings/dimension/minecraft:the_end match what a brand new one looks like. (I created a new world in singleplayer to pull the settings out of)
![level dat](https://user-images.githubusercontent.com/4063463/101245356-30306c80-36c1-11eb-93d1-3bf0938b68cd.png)



When I opened my servers `level.dat` that was started before 1.8.0 it had a ton of extra tags within it that didn't match a brand new worlds from 1.8.0 so I think that was the issue. 

It worked for me so hopefully it helps someone else out too.
